tests/test_loader.py

```diff
@@ -7,8 +6,7 @@
 from scrapy.http import HtmlResponse
 from scrapy.item import Item, Field
 from scrapy.loader import ItemLoader
-from scrapy.loader.processors import (Compose, Identity, Join,
-                                      MapCompose, SelectJmes, TakeFirst)
+from scrapy.loader.processors import Compose, Identity, MapCompose, TakeFirst
```

Comment:
In this and any other file were we import processors, we should start importing them from the new library.